sysdig/data_source_sysdig_fargate_workload_agent.go

```diff
@@ -146,11 +152,19 @@ type cfnStack struct {
 	Resources map[string]cfnResource `json:"Resources"`
 }
 
-// fargatePostKiltModifications performs any additional changes needed after
-// Kilt has applied it's transformations
-func fargatePostKiltModifications(patchedBytes []byte, logConfig map[string]interface{}) ([]byte, error) {
-	if len(logConfig) == 0 {
-		// no log configuration provided, nothing to do
+func contains(items []string, target string) bool {
```

Comment:
I am looking forward to proper slice support in Go 1.21 :-)

Reply:
yeah, I wanted to avoid adding a dependency (golang.org/x/exp/slices) to import such a simple function